src/eventHubMessageExplorer.ts

```diff
@@ -0,0 +1,90 @@
+        let eventHubPath = Utility.getConfig(Constants.EventHubPathKey, Constants.EventHubPathTitle);
+        if(!eventHubConnectionString || !eventHubPath) {
+            this.outputLine(Constants.EventHubMessageLabel, `Invalid arguments of ${ Constants.EventHub }.`);
+            return;
+        }
+        vscode.window.showInputBox({ prompt: `Enter message to send to  ${ Constants.EventHub }` }).then((message: string) => {
+            if (message !== undefined) {
+                let client = EventHubClient.fromConnectionString(eventHubConnectionString, eventHubPath);
+                try {
+                    let receiveAfterTime = Date.now() - 5000;
+                    client.open()
+                        .then(client.getPartitionIds.bind(client))
+                        .then(() => client.createSender())
+                        .then(this.sendToEventHub(message))
+                        .then(this.sendToEventHubDone(client));
+                }
+                catch (e) {
+                    this.sendToEventHubFail(client, e);
+                }
+            }
+        });
+    }
+
+    public startMonitorEventHubMessage(): void {
+        let eventHubConnectionString = Utility.getConfig(Constants.EventHubConnectionstringKey, Constants.EventHubConnectionStringTitle);
+        let eventHubPath = Utility.getConfig(Constants.EventHubPathKey, Constants.EventHubPathTitle);
+        if(!eventHubConnectionString || !eventHubPath) {
+            this.outputLine(Constants.EventHubMonitorLabel, `Invalid arguments of ${ Constants.EventHub }.`);
+            return;
+        }        
+        let config = Utility.getConfiguration();
+        let consumerGroup = config.get<string>(Constants.EventHubConsumerGroup);
+        try {
+            this._eventHubClient = EventHubClient.fromConnectionString(eventHubConnectionString, eventHubPath);
+            let receiveAfterTime = Date.now() - 5000;
+            this._outputChannel.show();
+            this.outputLine(Constants.EventHubMonitorLabel, `Start monitoring ${Constants.EventHub } ...`);
+            this._appInsightsClient.sendEvent(Constants.EventHubAIStartMonitorEvent);
+            this.startMonitor(this._eventHubClient, Constants.EventHubMonitorLabel, consumerGroup);
+        }
+        catch (e) {
+            this.outputLine(Constants.EventHubMonitorLabel, e);
+            this._appInsightsClient.sendEvent(Constants.EventHubAIStartMonitorEvent, {Result: 'Exception', Message: e})
+        }
+    }
+
+    public stopMonitorEventHubMessage(): void {
+        this.stopMonoitor(this._eventHubClient, Constants.EventHubMonitorLabel, Constants.EventHubAIStopMonitorEvent);
+        this._eventHubClient = null;
+    }
+
+    private sendToEventHub(eventBody: string) {
+        return (sender: EventHubSender) => {
+            this.outputLine(Constants.EventHubMessageLabel, `Sending message to ${Constants.EventHub } ...`);

```

Comment:
`client.open()` seems to take more time. Move this message before `client.open()` for better user experience